examples/issue/components/IssueEditor.js

```diff
@@ -36,7 +36,7 @@ export default class IssueEditor extends React.Component {
     };
   }
 
-  onChange = (editorState, cb) => this.setState({ editorState }, cb);
+  onChange = (editorState) => this.setState({ editorState });
```

Comment:
The [`drafts-js-plugins` docs for the editor props](https://github.com/draft-js-plugins/draft-js-plugins#editor) just refer to the [`draft-js` docs for `onChange`](https://facebook.github.io/draft-js/docs/api-reference-editor.html#onchange) which don't document a second parameter.